components/cam/bld/configure

```diff
@@ -277,6 +277,7 @@ OPTIONS
      -MMF_microphysics_scheme <string>   CRM microphysics package name [sam1mom | m2005 ].
      -use_ECPP          use CRM clouds for vertical transport, aqueous chemistry and wet removable of aerosols 
      -crm_adv           CRM advection scheme [MPDATA | UM5]
+     -crm <model>       CRM model [sam | samxx]
```

Comment:
Can you take a look at `components/cam/cime_config/testdefs/testmods_dirs/cam/crm_gpu/shell_commands`:
```
./xmlchange CAM_CONFIG_OPTS="-phys cam5 -use_MMF  -rad rrtmg -nlev 72 -microphys mg2 -crm_nz 58 
-crm_adv MPDATA -crm_dt 5 -crm_nx 64 -crm_ny 1 -crm_dx 1000 -crm_nx_rad 1 -crm_ny_rad 1 
-bc_dep_to_snow_updates -MMF_microphysics_scheme sam1mom -chem none -cppdefs ' -DSP_DIR_NS 
-DSP_MCICA_RAD ' "

./xmlchange ATM_NCPL=288
```

With the new `-crm $sam_model` addition, does that just need a `-crm sam` addition or something more?
Current configure error in `SMS_P36x1_Ld1.ne4pg2_ne4pg2.FC5AV1C-H01A.blues_pgigpu.cam-crm_gpu` is
```
Command: '/gpfs/fs1/home/e3smtest/jenkins/workspace/ACME_blues_next/E3SM/components/cam/bld/configure 
-s -ccsm_seq -ice none -ocn docn -caseroot /lcrc/group/acme/e3smtest/acme_scratch/blues/J/SMS_P36x1_Ld1.ne4pg2_ne4pg2.FC5AV1C-H01A.blues_pgigpu.cam-crm_gpu.JNextGpu20200913_235859 
-comp_intf mct  -spmd -spmd -smp -nosmp -dyn se -dyn_target preqx -res ne4np4.pg2   -phys cam5 
-use_MMF  -rad rrtmg -nlev 72 -microphys mg2 -crm_nz 58 -crm_adv MPDATA -crm_dt 5 -crm_nx 64 -crm_ny 1 
-crm_dx 1000 -crm_nx_rad 1 -crm_ny_rad 1 -bc_dep_to_snow_updates -MMF_microphysics_scheme sam1mom 
-chem none -cppdefs ' -DSP_DIR_NS -DSP_MCICA_RAD '  ' 
failed with error 'ERROR:  is not a valid value for parameter crm: valid values are sam,samxx' from dir 
'/lcrc/group/acme/e3smtest/acme_scratch/blues/J/SMS_P36x1_Ld1.ne4pg2_ne4pg2.FC5AV1C-H01A.blues_pgigpu.cam-crm_gpu.JNextGpu20200913_235859/Buildconf/camconf'
```

Reply:
@amametjanov that's an odd configuration you have there. You're using the hi-res compset with a low-res grid. Also, we changed the names of many variables, so like "SP_DIR_NS" doesn't exist anymore. I'm pretty sure the error is related to trying to manually combine the MMF options with the E3SM compset. Can you just use the "F-MMF1" compset that's currently in E3SM?